spec/models/post_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) { User.create(name: 'John') }
+  let(:subject_post) { Post.new(title: 'Welcome', author: user) }
+
+  describe 'validation tests' do
+    it 'title should be present' do
+      subject_post.title = nil
+      expect(subject_post).to_not be_valid
+    end
+
+    it 'title should be less than 251 chars' do
+      subject_post.title = 'Lorem ipsum dolor sit amet,
+      consectetuer adipiscing elit. Aenean commodo ligula
+      eget dolor. Aenean massa. Cum sociis natoque penatibus
+      et magnis dis parturient montes, nascetur ridiculus mus.
+      Donec quam felis, ultricies neln ea'
+      expect(subject_post).to_not be_valid
+    end
+
+    it 'comments_counter should be integer' do
+      subject_post.comments_counter = 'hi'
+      expect(subject_post).to_not be_valid
+    end
+
+    it 'comments_counter should be greater than or equal to zero' do
+      subject_post.comments_counter = -2
+      expect(subject_post).to_not be_valid
+      subject_post.comments_counter = 0
+      expect(subject_post).to be_valid
+    end
+
+    it 'likes_counter should be greater than or equal to zero' do
+      subject_post.likes_counter = -2
+      expect(subject_post).to_not be_valid
+      subject_post.likes_counter = 0
+      expect(subject_post).to be_valid
+    end
```

Comment:
- [ ] Great job working on the required changes but I think you still have some falling tests so I highly recommend solving them since Testing plays a vital role in the development of robust and reliable software. In the context of Ruby on Rails applications, having a solid testing strategy can prevent bugs, improve maintainability, and provide developers with the confidence to make changes and add new features without introducing regressions. so please work on them to be solved ✅✅

- Errors
![image](https://github.com/Simpleshaikh1/blog-app/assets/58553711/7fe3366e-f18f-48d9-ad24-4830845cf965)
![image](https://github.com/Simpleshaikh1/blog-app/assets/58553711/0e1605bd-70e3-4346-887b-4914d75c6fb8)
![image](https://github.com/Simpleshaikh1/blog-app/assets/58553711/a3ac17ba-4c39-4e0a-adcc-e47273f28662)
![image](https://github.com/Simpleshaikh1/blog-app/assets/58553711/0fd09e26-69a9-46d5-988d-0bcf569a44c5)